neurolib/utils/collections.py

```diff
@@ -26,9 +29,20 @@ def __setstate__(self, state):
 
 
 class star_dotdict(dotdict):
-    """Support star notation in dotdict. Nested dicts are now treated as glob"""
+    """
+    Supports star notation in dotdict. Nested dicts are now treated as glob.
+    Also supports minus as a pipe ("|") for filtering out strings after |.
+    """
```

Comment:
Very nice stuff. Could you add one or two examples here?

Reply:
The dotdict has become a pretty good object IMO :D

Reply:
I think we're pretty much good to go if you have time to add an example to the comments!